spec/models/volunteer_spec.rb

```diff
@@ -24,4 +24,31 @@
     expect(volunteer.errors[:name][0]).to eq("can't be blank")
     expect(volunteer.errors[:about][0]).to eq("can't be blank")
   end
+
+  describe "Volunteer groups" do
+    it "has collection of groups" do
+      volunteer_groups = Volunteer.groups
+      groups = {
+        "hospital" => "help nurses in hospitals",
+        "triage" => "visit patients in home isolation",
+        "healthcare" => "give care to dependents of healthcare workers",
+        "elderly" => "cater to the needs of the elderly",
+        "research" => "curate academic research for busy doctors",
+        "operations" => "operations",
+        "other" => "other"
+      }
+
+      expect(volunteer_groups).equal?(groups)
+    end
+
+    it "has a support_unit attribute" do
```

Comment:
Same comment with this test. I don't know that we really need to test that Rails created the method. And is the description for the test accurate? I don't see anything for `support_unit` here.

Reply:
Yes, I agree my mistake originally I was going to call the attribute `support_unit` I miss the change in this test